Documentation/network/standalone-load-balancer.rst

```diff
@@ -0,0 +1,285 @@
+    $ LB_VETH_HOST=$(ip -o l | grep "if$IFIDX" | awk '{print $2}' | cut -d@ -f1)
+    # ip l set dev $LB_VETH_HOST xdp obj bpf_xdp_veth_host.o
+    # ip l set dev l4lb-veth0 xdp obj bpf_xdp_veth_host.o
+
+Disable TX and RX checksum offload, as veth does not support it. Otherwise, the
+packets forwarded by the LB to the worker node will have invalid checksums:
+
+.. code-block:: shell-session
+
+    # ethtool -K $LB_VETH_HOST rx off tx off
+    # ethtool -K l4lb-veth0 rx off tx off
+
+Wait for the node to get ready:
+
+.. code-block:: shell-session
+
+    $ CILIUM_POD_NAME=$(kubectl -n kube-system get pod -l k8s-app=cilium -o=jsonpath='{.items[0].metadata.name}')
+    $ kubectl -n kube-system wait --for=condition=Ready pod "$CILIUM_POD_NAME" --timeout=5m
+
+Deploy the Load-Balancer
+------------------------
+
+.. code-block:: shell-session
+
+    $ ./daemon/cilium-agent \
+        --enable-ipv4=true \
+        --enable-ipv6=true \
+        --datapath-mode=lb-only \
+        --bpf-lb-algorithm=maglev \
+        --bpf-lb-maglev-table-size=2039 \
+        --bpf-lb-mode=dsr \
+        --bpf-lb-acceleration=native \
+        --devices=enp2s0np0 \
+        --bpf-lb-dsr-dispatch=ipip \
+        --disable-envoy-version-check=true
+
+Validate the Setup
+------------------
+
+We retrieve the LB node's IP address, and set the relevant routes for the
+traffic:
+
+.. code-block:: shell-session
+
+    $ LB_VIP="10.0.0.2"
+    # nsenter -t $(docker inspect kind-worker -f '{{ .State.Pid }}') -n /bin/sh -c \
+        "ip a a dev eth0 ${LB_VIP}/32"
+    $ LB_NODE_IP=$(docker exec kind-control-plane ip -o -4 a s eth0 | awk '{print $4}' | cut -d/ -f1)
+    # ip r a "${LB_VIP}/32" via "$LB_NODE_IP"
+
+Then we declare the LB service (NodePort), using the LB node as a frontend, and
+the Nginx worker nodes as a backend, listening on port 80:
+
+    $ kubectl -n kube-system exec "${CILIUM_POD_NAME}" -- \
+        cilium service update --id 1 --frontend "${LB_VIP}:80" --backends "${WORKER_IP}:80" --k8s-node-port
+
+We can issue a few requests to the LB, and observe they all go through:
+
+.. code-block:: bash
+
+    for i in $(seq 1 10); do
+        curl -o /dev/null "${LB_VIP}:80" || (echo "Failed $i"; exit -1)
+    done
+
+Now we steer the traffic to ``LB_VIP`` via the secondary device so that
+``XDP_REDIRECT`` can be tested on the LB node:
+
+.. code-block:: shell-session
+
+    # ip r replace "${LB_VIP}/32" via "$SECOND_LB_NODE_IP"
+
+Again, we send some requests to the LB and should see them answered:
+
+.. code-block:: bash
+
+    for i in $(seq 1 10); do
+        curl -o /dev/null "${LB_VIP}:80" || (echo "Failed $i"; exit -1)
+    done
+
+Now we set ``kind-worker`` to maintenance, by assigning a weight of ``0`` to
+the backend:
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system exec "${CILIUM_POD_NAME}" -- \
+        cilium service update --id 1 --frontend "${LB_VIP}:80" --backends "${WORKER_IP}:80" --backend-weights "0" --k8s-node-port
```

Comment:
small note, I would maybe add a second option to use the update of backend states to set worker (shared across all services) to maintenance?..
`cilium service update --backends "${WORKER_IP}:80" --k8s-node-port --states maintenance`